paymentForm.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Payment Form</title>
+<p>
+<h1>Payment Form</h1>
+</p>
+</head>
+
+<body>
+<p>
+Required fields are marked by <u>*</u>
+</p>
+<h3>Contact Information</h3>
+<input type="radio" name="title" value="Mr.">Mr.<br>
```

Comment:
Remember to close your input tag with `/>` this will throw errors in the future.